src/giterm/cursutils.py

```diff
@@ -40,7 +40,7 @@ def debug(stdscr=None):
 
 
 def log(msg):
-    with open('../giterm.log', 'a') as f:
+    with open('giterm.log', 'a') as f:
```

Comment:
Historically giterm was launch from inside the working copy so this was to prevent constant File Change Events to happen while logging...
Doing so may raise such loops back again. I suggest to let it as it was before until we change this behavior.